public/configurations/queries/inf-regress-express-lastrun-heatmap.json

```diff
@@ -0,0 +1,132 @@
+{

```

Comment:
Is all the delta between this file and https://github.com/nuagenetworks/visualization-framework/blob/nuage-infra/public/configurations/queries/inf-regress-area-testsuite-heatmap.json valid?

Reply:
The filtered suites are different in both, so we should do it at the elasticsearch level